### DISCUSSION TITLE
Ignore Current Update

### DISCUSSION THREAD

**Discussion by gerbil-snot** [wstyres/Zebra#1336] [Ideas]
**Is your feature request related to a problem? Please describe.**
It can be annoying when a recently updated tweak becomes broken, then after you downgrade it, every time you go to update other tweaks in bulk you end up updating the broken version of said tweak. You still want to update it, just not until the issues been fixed. 

**Describe the solution you'd like**
An extra option when ignoring updates for a tweak to “Ignore current update” for those tweaks that may be broken on the current version or for the tweaks that just won’t update. This feature would temporarily hide updates for the tweak until a new version is released. 

**Describe alternatives you've considered**
An alternative could be to “Ignore updates for x amount of time”

**Comment by blanxd** [comment]
this would be helpful for beta testers and other such cases! (might even eg. decide to downgrade Zebra beta at the current moment, and be curious about any next beta iterations :)
